cli/Dockerfile-bin

```diff
@@ -1,5 +1,5 @@
 ## compile binaries
-FROM gcr.io/runconduit/go-deps:f24620b6 as golang
+FROM gcr.io/runconduit/go-deps:e7e2dd08 as golang
```

Comment:
on my branch i see `eb0dd1cd` as the tag:
```bash
$ . bin/_tag.sh ; go_deps_sha
eb0dd1cd
```

Reply:
> on my branch i see eb0dd1cd as the tag:

It probably got out of sync during rebasing. I'll rebase and run bin/update-go-deps-shas right before I merge this.